app/src/main/java/com/isc/hermes/model/User.java

```diff
@@ -0,0 +1,202 @@
+     */
+    @Override
+    public int describeContents() {
+        return 0;
+    }
+
+    /**
+     * Creator for the User class, which creates a User object from a Parcel.
+     */
+    public static final Creator<User> CREATOR = new Creator<>() {
+        @Override
+        public User createFromParcel(Parcel in) {
+            return new User(in);
+        }
+
+        @Override
+        public User[] newArray(int size) {
+            return new User[size];
+        }
+    };
+
+    /**
+     * Returns the full name of the user.
+     *
+     * @return The full name of the user.
+     */
+    public String getFullName() {
+        return fullName;
+    }
+
+    /**
+     * Returns the email of the user.
+     *
+     * @return The email of the user.
+     */
+    public String getEmail() {
+        return email;
+    }
+
+    /**
+     * Returns the username of the user.
+     *
+     * @return The username of the user.
+     */
+    public String getUserName() {
+        return userName;
+    }
+
+    /**
+     * Returns the path to the user's image.
+     *
+     * @return The path to the user's image.
+     */
+    public String getPathImageUser() {
+        return pathImageUser;
+    }
+
+    /**
+     * Returns the ID of the user.
+     *
+     * @return The ID of the user.
+     */
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * Returns the type of the user.
+     *
+     * @return The type of the user.
+     */
+    public String getTypeUser() {
+        return typeUser;
+    }
+
+    /**
+     * Sets the type of the user.
+     *
+     * @param typeUser The type of the user.
+     */
+    public void setTypeUser(String typeUser) {
+        this.typeUser = typeUser;
+    }
+
+
+    /**
+     * Sets the username of the user.
+     *
+     * @param userName The username of the user.
+     */
+    public void setUserName(String userName) {
+        this.userName = getFirstWord(userName);
+    }
+
+    /**
+     * Sets the full name of the user.
+     *
+     * @param name The first name of the user.
+     * @param lastName The last name of the user.
+     */
+    public void setFullName(String name, String lastName){
+        StringBuilder str = new StringBuilder();
+        str.append(getFirstWord(name));
+        if (lastName != null) str.append(" ").append(getFirstWord(lastName));
+        this.fullName = str.toString();
+    }
+
+    /**
+     * Returns a string representation of the `User` object.
+     *
+     * @return A string representation of the `User` object.
+     */
+    @Override
+    public String toString() {
+        return "User{" +
+                "fullName='" + fullName + '\'' +
+                ", email='" + email + '\'' +
+                ", userName='" + userName + '\'' +
+                ", pathImageUser='" + pathImageUser + '\'' +
+                ", typeUser='" + typeUser + '\'' +
+                ", id='" + id + '\'' +
+                '}';
+    }
```

Comment:
Delete this method, looks like its purposes were only for testing.

Reply:
Done